components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/service/api/GroupManagementService.java

```diff
@@ -672,8 +732,8 @@ Response addDevicesToGroup(@ApiParam(
                                        required = true)
                                @Valid List<DeviceIdentifier> deviceIdentifiers);
 
-    @Path("/id/{groupId}/devices")
-    @DELETE
+    @Path("/id/{groupId}/devices/remove")
```

Comment:
This operation is invoked to remove set of devices from a group. So that a list of device identifiers should be passed with the request body and since DELETE method doesn't use request body, it is decided to use POST instead.